kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/model/VirtualCluster.java

```diff
@@ -229,6 +259,50 @@ private static NettyTrustProvider configureTrustProvider(Tls tlsConfiguration) {
         return new NettyTrustProvider(trustProvider);
     }
 
+    private static void configureCipherSuites(SslContextBuilder sslContextBuilder, Tls tlsConfiguration) {
+        Optional.ofNullable(tlsConfiguration.cipherSuites())
+                .map(ciphers -> sslContextBuilder.ciphers(
+                        tlsConfiguration.cipherSuites().allowed(),
+                        new DenyCipherSuiteFilter(tlsConfiguration.cipherSuites().denied())));
+    }
+
+    private static void configureEnabledProtocols(SslContextBuilder sslContextBuilder, Tls tlsConfiguration) {
+
+        try {
+            var protocols = Optional.ofNullable(tlsConfiguration.protocols());
+
+            if (protocols.isPresent()) {
+                var allowedProtocols = Optional.ofNullable(protocols.get().allowed())
+                        .orElse(Arrays.stream(SSLContext.getDefault().getSupportedSSLParameters().getProtocols())
```

Comment:
I _think_ we want to base on the default ssl parameters rather than the platform 
rather than the supported.  The platform might support an aged `SSLv1`, but it is turned off by default.  We don't want to inadvertently enable historic stuff.



```suggestion
                        .orElse(Arrays.stream(SSLContext.getDefault().getDefaultSSLParameters().getProtocols())
```